examples/example_for_updating_metadata.py

```diff
@@ -53,29 +28,58 @@
         "https://orcid.org/0000-0001-8176-199X",
         row_name='Contributor orcid')
 
-    dataset_description_editor.add_values(*["University of Auckland"]*6, row_name='Contributor affiliation')
-    dataset_description_editor.add_values(*["developer", "developer", "Researcher", "Researcher", "tester", "tester"], row_name="contributor role")
+    dataset_description_editor.add_values(*["University of Auckland"] * 6, row_name='Contributor affiliation')
+    dataset_description_editor.add_values(*["developer", "developer", "Researcher", "Researcher", "tester", "tester"],
+                                          row_name="contributor role")
     dataset_description_editor.add_values("source", row_name='Identifier description')
     dataset_description_editor.add_values("WasDerivedFrom", row_name='Relation type')
     dataset_description_editor.add_values("DTP-UUID", row_name='Identifier')
     dataset_description_editor.add_values("12L digital twin UUID", row_name='Identifier type')
     dataset_description_editor.add_values("1", row_name='Number of subjects')
     dataset_description_editor.add_values("1", row_name='Number of samples')
 
+if __name__ == '__main__':
+    save_dir = "./tmp/template/"
+
+    dataset = Dataset()
+
+    # Step:1 list categories and dataset_description elements
+    categories = dataset.list_categories(version="2.0.0")
+    elements = dataset.list_elements(category="dataset_description", version="2.0.0")
+    # list code_parameters elements
+    # elements = dataset.list_elements(category="code_parameters", version="2.0.0")
+
+    # Step2, way1: load dataset from template
+    dataset.load_from_template(version="2.0.0")
+    #
+    # Save the template dataset.
+    # dataset.save(save_dir=save_dir)
+
+    # Step2, way2： load dataset from existing dataset
+    # dataset.load_dataset(dataset_path=save_dir)
+
+    # Step3, get dataset_description, code_description, code_parameters metadataEditor
+    dataset_description_editor = dataset.get_metadata(category="dataset_description")
+    code_parameters_editor = dataset.get_metadata(category="code_parameters")
+    code_description_editor = dataset.get_metadata(category="code_description")
+
```

Comment:
remove `editor `from var names

Reply:
fixed by [8568637](https://github.com/SPARC-FAIR-Codeathon/sparc-me/pull/92/commits/856863752c90fad72015559f05f71db91768eca1) and [ebdf368](https://github.com/SPARC-FAIR-Codeathon/sparc-me/pull/92/commits/ebdf368249b491903495b95c6db6e8289dd8e9d3)